tests/scripts/task_config_build_gpu.sh

```diff
@@ -41,7 +41,6 @@ echo set\(USE_PROFILER ON\) >> config.cmake
 echo set\(USE_ANTLR ON\) >> config.cmake
 echo set\(USE_VTA_FSIM ON\) >> config.cmake
 echo set\(USE_BLAS openblas\) >> config.cmake
-echo set\(CMAKE_CXX_COMPILER g++\) >> config.cmake
```

Comment:
i think removing this should be fine, but has this been tested locally outside the CI?

Reply:
Yeah I tested, by default it will use `cc`/`c++` as the compiler which usually aliases to gcc anyways